src/pages/Network.vue

```diff
@@ -66,17 +115,44 @@ export default {
     ])
   },
   methods: {
-    update (newVal) {
-      if (this.current === newVal) return
-      this.current = newVal
-      let option = this.options[this.current]
-      console.log('should update to', option)
-      this.$store.commit('setNodeSettings', option.nodeSettings)
+    updateNetwork (newVal) {
+      this.$store.commit('setNodeSettings', newVal.nodeSettings)
+    },
+    removeNetwork (index) {
+      this.options.splice(index, 1)
+      this.saveNetworks()
+      console.log(index)
+    },
+    saveNetworks () {
+      localStorage.setItem(
+        'networks',
+        JSON.stringify(this.options)
+      )
```

Comment:
We are using `vuex-persistedstate` plugin for synchronization between Vuex store and local storage, why don't use it in this case instead of manual synchronization with local storage?
https://github.com/aeternity/aepp-identity/blob/97a7544/src/store/index.js#L26-L28